test/test-list.h

```diff
@@ -165,7 +165,9 @@ TEST_DECLARE   (idle_starvation)
 TEST_DECLARE   (loop_handles)
 TEST_DECLARE   (get_loadavg)
 TEST_DECLARE   (walk_handles)
+#ifndef __MVS__
```

Comment:
Weren't you previously skipping this in the actual test? I think that would be preferable, with a skip message that doesn't reference an internal IBM ticket.